modules/gitops-creating-a-new-client-in-dex.adoc

```diff
@@ -0,0 +1,31 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+= Enabling the Dex OpenShift OAuth Connector
+
+Dex uses the users and groups defined within OpenShift by checking the platform provided `OAuth` server. See the following properties of Dex:
```

Comment:
```suggestion
Dex uses the users and groups defined within OpenShift by checking the `OAuth` server provided by the platform. See the following properties of Dex:
```
- The 1st sentence has awkward syntax. Reword.
- The second sentence should be rephrased. It doesn't really properly introduce the bulleted list because a user can't `see` properties. What do you want to communicate to users about the list? Maybe something like `You can configure the following properties of Dex:` or something similar.